test/test_init.py

```diff
@@ -163,3 +164,109 @@ def test_shock_init(ctx_factory):
     p = eos.pressure(cv)
 
     assert discr.norm(actx.np.where(nodes_x < 0.5, p-xpl, p-xpr), np.inf) < tol
+
+
+@pytest.mark.parametrize("dim", [1, 2, 3])
+def test_uniform(ctx_factory, dim):
```

Comment:
It doesn't look like the new BC is being tested at all.

Reply:
Added some tests for slip wall.